src/index.html

```diff
@@ -17,11 +17,379 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="/src/styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="header__logo"
+      >
+        <img
+          class="header__logo-moyo"
+          src="/src/images/logo.png"
+          alt="Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="is-active nav__item-button"
+              href="/"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__item-button"
+              chref="/"
```

Comment:
There is a typo in the `href` attribute of this link. It is currently written as `chref`, which will cause the link to not function correctly. Please correct it to `href`.